Example/PaymentSheet Example/PaymentSheet Example/ExampleLinkPaymentCheckoutViewController.swift

```diff
@@ -27,8 +27,8 @@ class ExampleLinkPaymentCheckoutViewController: UIViewController {
     let billingDetails: PaymentSheet.BillingDetails = {
         var billingDetails = PaymentSheet.BillingDetails()
         // uncomment to test prefilled email
-        // billingDetails.email = "test@example.com"
-        // billingDetails.phone = "+15551232414567"
+        billingDetails.email = "email_\(UUID().uuidString)@email.com"
+        billingDetails.phone = "+15551232414567"
```

Comment:
Is this intentionally left un-commented?

Reply:
I commented it out again.